<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>448</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>405</width>
    <height>650</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>405</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>My Notes</string>
  </property>
  <widget class="QComboBox" name="noteItem">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="noteText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>361</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveNote">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save Note</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
